src/main/java/uk/gov/pay/api/model/search/directdebit/DirectDebitSearchPaymentsParams.java

```diff
@@ -34,11 +33,9 @@
     private String mandateId;
     
     @QueryParam("from_date")
-    @ValidDate
```

Comment:
Why have these annotations gone?

Reply:
It was a mistake, I've put them back in now.